src/java/org/apache/cassandra/tcm/CMSOperations.java

```diff
@@ -243,4 +244,16 @@ public void unregisterLeftNodes(List<String> nodeIdStrings)
             cms.commit(new Unregister(nodeId, EnumSet.of(NodeState.LEFT)));
         }
     }
+
+    @Override
+    public void accordMarkStale(List<String> nodeIdStrings)
```

Comment:
to be clear on intent, this is to "append" and not "override" the current set right? Append I think is the common case, not sure if we want to have a "YOLO I KNOW WHAT I AM DOING; OVERRIDE!"

Reply:
Right, "mark" == "append" and "unmark" will be "remove" (and both will take lists shortly...)